requirements.txt

```diff
@@ -2,3 +2,5 @@ django
 django-extensions
 django-bootstrap3
 psycopg2
+numpy
```

Comment:
numpy is a big package, consider removing it since workalendar already provides this feature and I suggested some changes where you won't be using numpy.